integration-test/src/main/java/com/sequenceiq/it/cloudbreak/actor/CloudbreakUserCache.java

```diff
@@ -89,30 +92,49 @@ public CloudbreakUser getByDisplayName(String name) {
         if (MapUtils.isEmpty(usersByAccount)) {
             initRealUmsUserCache();
         }
-        CloudbreakUser user = usersByAccount.values().stream().flatMap(Collection::stream)
-                .filter(u -> u.getDisplayName().equals(name)).findFirst()
-                .orElseThrow(() -> new TestFailException(String.format("There is no real UMS user with::%n name: %s%n deployment::account: %s::%s", name,
-                        realUmsUserDeployment, realUmsUserAccount)));
-        LOGGER.info(" Real UMS user has been found:: \nDisplay name: {} \nCrn: {} \nAccess key: {} \nSecret key: {} \nAdmin: {} ",
-                user.getDisplayName(), user.getCrn(), user.getAccessKey(), user.getSecretKey(), user.getAdmin());
-        return user;
+        if (MapUtils.isNotEmpty(usersByAccount)) {
+            CloudbreakUser user = usersByAccount.values().stream().flatMap(Collection::stream)
+                    .filter(u -> u.getDisplayName().equals(name)).findFirst()
+                    .orElseThrow(() -> new TestFailException(String.format("There is no real UMS user with::%n name: %s%n deployment::account: %s::%s", name,
+                            realUmsUserDeployment, realUmsUserAccount)));
+            LOGGER.info(" Real UMS user has been found:: \nDisplay name: {} \nCrn: {} \nAccess key: {} \nSecret key: {} \nAdmin: {} ",
+                    user.getDisplayName(), user.getCrn(), user.getAccessKey(), user.getSecretKey(), user.getAdmin());
+            return user;
+        } else {
+            throw new TestFailException("UMS user store 'ums-users/api-credentials.json' is not available." +
```

Comment:
the message should be something related to the method, like: "Getting user by name ... failed since ums user store is empty"